requirements.txt

```diff
@@ -4,6 +4,7 @@ PyYAML
 repoze.lru
 pbkdf2
 pycryptodome>=3.3.1
+future==0.16.0
```

Comment:
Once I change it to `itertools.count` there won't be any use for `future`. I was using it to avoid memory errors in Python2 when using `range()` which thankfully I won't need anymore